nemo/collections/cv/modules/__init__.py

```diff
@@ -0,0 +1,10 @@
+from nemo.collections.cv.modules.conv2d import Conv2d
```

Comment:
We use the `modules` dir for NeuralModules mainly - ie that which can be imported directly one level above in a ModelPT subclass or can be instantiated by hydra in a config. They should also have their input-output types defined because they are directly called and checked by the actual classes (if provided) 

So utils like permute, GAP, ResNetBlock, ResNetBottleneck should go in a `cv/parts` directory. They may or may not inherit torch.nn.Module or NeuralModule, but only primary components used inside Model should be inside Module - not the building blocks.